libs/core/src/lib/shellbar/user-actions-menu/user-actions-menu-item/user-actions-menu-item.component.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostListener,
+  Input,
+  Output,
+  ViewEncapsulation,
+  EventEmitter,
+  Host,
+  OnDestroy
+} from '@angular/core';
+import { take } from 'rxjs/operators';
+
+import { DialogContentType, DialogRef, DialogService } from '../../../dialog/public_api';
+import { UserActionsMenuComponent } from '../user-actions-menu/user-actions-menu.component';
+
+@Component({
+  selector: 'fd-user-actions-menu-item',
+  templateUrl: './user-actions-menu-item.component.html',
+  styleUrls: ['./user-actions-menu-item.component.scss'],
+  host: {
+    '[class.fd-user-actions-menu-item]': 'true',
+    '[attr.tabindex]': 'tabIndex'
+  },
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class UserActionsMenuItemComponent implements OnDestroy {
+  @Input()
+  title: string;
+
+  @Input()
+  glyph: string;
+
+  @Input()
+  glyphSrc: string;
+
+  @Input()
+  routerLink: string | any[];
+
+  @Input()
+  dialogContent: DialogContentType;
```

Comment:
I see it works well, but I'm wondering if there is any sense of having this here, instead of just opening dialog by application side.

Reply:
I added it only for simple content, for example, `About` or something like this. Without any business logic or functionality into them. But of course, you as a developer can add click handler and add any functionality to this item.